pkg/az/az_test.go

```diff
@@ -77,7 +71,7 @@ func cleanupTestServicePrincipal(spObjectID, appObjectID string, t *testing.T) {
 // func TestShouldSuccessfullyCreateServicePrincipal(t *testing.T) {
 
 // 	// when
-// 	pass := GenerateServicePrincipalPassword()
+// 	pass := GeneratePassword()
 
```

Comment:
You can uncomment those tests. It was done only for avoiding issue with SP.

Reply:
I will do it when I reach this part of tests.